lib/rb-kqueue/watcher/file.rb

```diff
@@ -42,7 +42,7 @@ def initialize(queue, path, flags, callback)
       end
 
       def self.finalizer(fd, path)
-        lambda do
+        lambda do |id|
```

Comment:
Would this break if this lambda is called with no arguments? I assume so, given that this works in my environment as-is. Perhaps we should have a default value to prevent that case?

```suggestion
        lambda do |id=nil|
```

Reply:
I've made it optional, thanks.